interface/reports/insurance.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Financial Report initial page
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once '../../vendor/autoload.php';
```

Comment:
hi @juggernautsei , shouldn't need `/vendor/autoload.php` since it's run once when the application starts i believe

Reply:
I removed the autoload and it works fine.